modules/notification/action/action.go

```diff
@@ -191,3 +192,52 @@ func (a *actionNotifier) NotifyPullRequestReview(pr *models.PullRequest, review
 		log.Error("notify watchers '%d/%d': %v", review.Reviewer.ID, review.Issue.RepoID, err)
 	}
 }
+
+func (a *actionNotifier) NotifySyncPushCommits(pusher *models.User, repo *models.Repository, refName, oldCommitID, newCommitID string, commits *models.PushCommits) {
+	data, err := json.Marshal(commits)
+	if err != nil {
+		log.Error("json.Marshal: %v", err)
+		return
+	}
+
+	if err := models.NotifyWatchers(&models.Action{
```

Comment:
I wish this could be deactivated by the user. My dashboard has more updates from mirrors than my company's projects. 🙄

Reply:
Just not watch that repositories?

Reply:
> 
> 
> Just not watch that repositories?

🤦‍♂ 👍